src/components/ReportActionItem/MoneyRequestView.js

```diff
@@ -108,15 +110,17 @@ function MoneyRequestView({report, parentReport, shouldShowHorizontalRule, trans
             </View>
 
             {hasReceipt && (
-                <View style={styles.moneyRequestViewImage}>
-                    <ReportActionItemImage
-                        thumbnail={receiptURIs.thumbnail}
-                        image={receiptURIs.image}
-                        enablePreviewModal
-                    />
-                </View>
+                <OfflineWithFeedback pendingAction={pendingAction}>
```

Comment:
Passing the `pendingAction` value directly as `pendingAction` to ReportActionItemImage causes the opacity to reduce to 0.25 when parentReportAction.pendingAction is set to 'add'.

Instead, we should pass the specific keys for pendingAction—pendingAction.waypoints and waypoints.receipt.

More context can be found in the selected proposal: https://github.com/Expensify/App/issues/52123#issuecomment-2460598355